homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -65,6 +80,10 @@ async def async_step_bluetooth(
         if not _is_supported(discovery_info):
             return self.async_abort(reason="no_devices_found")
 
+        self.context["title_placeholders"] = {
+            "name": MANUFACTURER,
```

Comment:
@alistair23 is it possible to get the device name at this early stage (without pin)?